client-src/elements/chromedash-form-field.js

```diff
@@ -44,6 +45,7 @@ export class ChromedashFormField extends LitElement {
   getValue() {
     // value can be a js or python boolean value converted to a string
     // or the initial value specified in form-field-spec
+    // If value is falsy, it will be replaced by the initial value, if any.
```

Comment:
What is the difference between this code and:
return this.value || this.fieldProps.initial;
?
Is the goal to keep this.value when they are both falsy?   I am not sure that your comment is correct in that case.

Reply:
If this.value is a falsy value like 0 or '', then `!this.value` will be true.  And then, if this.fieldProps.initial is truthy, the whole condition `!this.value && this.fieldProps.initial` will be true, in which case the result of getValue will be this.fieldProps.initial.  That's what I meant by my comment, which I believe is correct.

To answer your question about `return this.value || this.fieldProps.initial`, it will have a similar but not identical behavior.  If this.value is falsy, then this.fildProps.initial will be used regardless whether the initial value is truthy.  

So, the fact that any falsy value (0, '', false) will be treated the same as null or undefined means that non-null values that are falsy will still be replaced by the initial value.  This means we can't store falsy values that differ from the initial value.  I would think we should allow 0 or false to be stored, and not be replaced by the initial value.  I was trying to make a change to check explicitly for null or undefined, with `this.value == null`, but that conflicted with something I don't recall at the moment.

Reply:
Thanks for explaining.  It's a subtle bit of logic, but I can see why it is needed.